src/Tools/babylon.dds.ts

```diff
@@ -517,13 +517,18 @@
                             engine._uploadCompressedDataToTexture(sampler, i, internalFormat, width, height, byteArray);
                         }
                     }
-                    dataOffset += width * height * (bpp / 8);
+                    dataOffset += dataLength;

```

Comment:
Actually this does not work for non compressed DDS
I'll push a fix to update to:
dataOffset += bpp ? (width * height * (bpp / 8)) : dataLength;

Reply:
I never knew there was such a thing as non-compressed DDS :)

Reply:
Pretty useful as compressed DDS is not wide supported